docs/core/diagnostics/memory-analysis.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Memory Analysis
+description: An introduction to memory analysis in .NET
+ms.date: 12/13/2022
+---
+
+# Memory Analysis
+
+Various behaviors can be manifested by your app as a result of a memory leak. The most common are crashing, possibly with an out of memory exception, decrease of performance, consuming large amounts of cloud storage. The method to verify a memory leak as the problem is to investigate the memory usage of your app. Memory usage can be investigated at one specific point in time, or multiple points of time to show the change in memory use. For more information see [debug memory leak tutorial](debug-memory-leak.md).
```

Comment:
If the goal is just to address issue #2905 I don't think we need to add this new Memory Analysis page. If we are primarily going to talk about memory leaks then hopefully we could do that by editing the existing tutorial page for diagnosing memory leaks?